local-devnet/genesis/validator-config.yaml

```diff
@@ -1,9 +1,14 @@
 shuffle: roundrobin
+config:
+  activeEpoch: 18
 validators:
   - name: "zeam_0"
     # node id 7d0904dc6d8d7130e0e68d5d3175d0c3cf470f8725f67bd8320882f5b9753cc0
     # peer id 16Uiu2HAkvi2sxT75Bpq1c7yV2FjnSQJJ432d6jeshbmfdJss1i6f
     privkey: "bdf953adc161873ba026330c56450453f582e3c4ee6cb713644794bcfdd85fe5"
+    # Hash-sig validator key configuration
+    keyType: "hash-sig" # Use hash-based signatures for post-quantum security
+    hashSigKeyIndex: 0 # Index into generated hash-sig keys
```

Comment:
why are you assiging the index here, the output file validators.yaml generated by PK's genesis generator already assigns index